app/models/container_replicator.rb

```diff
@@ -8,8 +8,18 @@ class ContainerReplicator < ActiveRecord::Base
   has_many :labels, -> { where(:section => "labels") }, :class_name => "CustomAttribute", :as => :resource, :dependent => :destroy
   has_many :selector_parts, -> { where(:section => "selectors") }, :class_name => "CustomAttribute", :as => :resource, :dependent => :destroy
   has_many :container_nodes, -> { distinct }, :through => :container_groups
+  has_many :metric_rollups, :as => :resource
+
+  # Needed for metrics
+  has_many :metrics,                :as => :resource
+  has_many :metric_rollups,         :as => :resource
+  has_many :vim_performance_states, :as => :resource
+  delegate :my_zone,                :to => :ext_management_system
```

Comment:
@abonas Its changes from #5338 that I needed here.
However #5587 was merged so I no longer need to be rebased on #5338.
Ill push a properly based version.